extrapolation.py

```diff
@@ -1421,6 +1432,12 @@ def _Extrapolate(FiniteRadiusWaveforms, Radii, ExtrapolationOrders, Omegas=None)
     NFiniteRadii = len(FiniteRadiusWaveforms)
     NExtrapolations = len(ExtrapolationOrders)
     SVDTol = 1.0e-12  # Same as Numerical Recipes default in fitsvd.h
+    DataType = FiniteRadiusWaveforms[NFiniteRadii-1].dataType
+    ExcludeInsignificantRadii = True if ((DataType in [scri.psi1, scri.psi0]) and ErrorTol) else False
```

Comment:
```suggestion
    ExcludeInsignificantRadii = DataType in [scri.psi1, scri.psi0]) and bool(ErrorTol)
```
Note that `ErrorTol is not None` is the usual test, but `bool(ErrorTol)` also handles cases where `ErrorTol==0`.

Reply:
Thanks for the idea about the bool cast, I hadn't thought of that.